sigsegv when there is no package

AFAIK, package is not mandatory in the protobuf file.

I want to create a service without a package name, I would end up with URLs like:
```
/twirp/Service/Method
```
No problem with twirp, generated Go code is okay... But there is a sigsegv error when I want to generate a swagger file:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x5e9bf4]

goroutine 1 [running]:
github.com/elliots/protoc-gen-twirp_swagger/genswagger.applyTemplate(0xc4200ea800, 0xc420103570, 0xd, 0xc42007fc48, 0x1, 0x1)
	/home/sebastien/Code/go/src/github.com/elliots/protoc-gen-twirp_swagger/genswagger/template.go:631 +0x294
github.com/elliots/protoc-gen-twirp_swagger/genswagger.(*generator).Generate(0xc42000e0c8, 0xc42000e0d0, 0x1, 0x1, 0x1, 0xc42000e0d0, 0x0, 0x1, 0x0)
	/home/sebastien/Code/go/src/github.com/elliots/protoc-gen-twirp_swagger/genswagger/generator.go:35 +0x184
main.main()
	/home/sebastien/Code/go/src/github.com/elliots/protoc-gen-twirp_swagger/main.go:84 +0x61a
--twirp_swagger_out: protoc-gen-twirp_swagger: Plugin failed with status code 2.
```

Line genswagger/template.go:631 is:
```
if err := renderServices(*p.Package, p.Services, s.Paths, p.reg, refs); err != nil {
```

Looks like `*p.Package` is a nil pointer when there is no package...

--- a/genswagger/template.go
+++ b/genswagger/template.go
@@ -506,8 +506,12 @@ func renderServices(pkg string, services []*descriptor.Service, paths swaggerPat
 	// Correctness of svcIdx and methIdx depends on 'services' containing the services in the same order as the 'file.Service' array.
 	for svcIdx, svc := range services {
 		for methIdx, meth := range svc.Methods {
-
-			path := fmt.Sprintf("/twirp/%s.%s/%s", pkg, *svc.Name, *meth.Name)
+			var path string
+			if pkg == "" {
+				path = fmt.Sprintf("/twirp/%s/%s", *svc.Name, *meth.Name)
+			} else {
+				path = fmt.Sprintf("/twirp/%s.%s/%s", pkg, *svc.Name, *meth.Name)
+			}
 
 			parameters := swaggerParametersObject{}
 
@@ -628,7 +632,7 @@ func applyTemplate(p param) (string, error) {
 	// Loops through all the services and their exposed GET/POST/PUT/DELETE definitions
 	// and create entries for all of them.
 	refs := refMap{}
-	if err := renderServices(*p.Package, p.Services, s.Paths, p.reg, refs); err != nil {
+	if err := renderServices(p.GetPackage(), p.Services, s.Paths, p.reg, refs); err != nil {
 		panic(err)
 	}
 

